.gitignore

```diff
@@ -204,3 +204,6 @@ FakesAssemblies/
 
 # Visual Studio 6 workspace options file
 *.opt
+
+# Project files
+IVRPhoneTree.Web/Scripts/lib/
```

Comment:
Visual Studio will complain if there is no `lib` folder already present, so I suggest creating the folder and commiting to the repo with a `.gitkeep` file.

Here the entry can be like this:
```
# Libman
**/Scripts/lib/
!.gitkeep
```